### ISSUE TITLE
Any way to change smart plug "In Use" threshold?

### ISSUE THREAD

**Issue by tamir61** [nitaybz/homebridge-switcher-platform#27]
Basically, I have a consumer that uses the smart plug and while its on sleep mode sometimes I'm getting InUse=true, maybe because of the power supply drawing higher current on some random points. i belive its drawing less than its on.
this isUse value effect my automation. I'm getting false positive. 

so my if I'll be able to change the threshold it will be great.
btw I'm using Eve app

Thanks!

**Comment by nitaybz** [comment]
That's actually a good idea...
for now, it's recommended to use it in parallel with homebridge-delay-switch

**Comment by tamir61** [comment]
mmmmmm.... I'm trying to understand how can It be useful in my case... What did you think? 🤔

**Comment by tamir61** [comment]
Yeah, it fixed the issue using homebridge-delay-switch.. but added more automations..
